io.catenax.request_for_quotation/1.0.0/RequestForQuotation.ttl

```diff
@@ -0,0 +1,584 @@
+   samm:description "RFQ-Configuration: placeholder for any kind of additional comments"@en ;
+   samm:characteristic :CommentCharacteristic .
+
+:additionalFiles a samm:Property ;
+   samm:preferredName "additionalFiles"@en ;
+   samm:description "RFQ-Configuration: in case there are additional documents or files to share"@en ;
+   samm:characteristic :SetOfFilesCharacteristic .
+
+:senderName a samm:Property ;
+   samm:preferredName "sender name"@en ;
+   samm:description "RFQ-Contact: Name of the sender"@en ;
+   samm:characteristic :SenderNameCharacteristic ;
+   samm:exampleValue "John Doe" .
+
+:senderAdress a samm:Property ;
+   samm:preferredName "sender adress"@en ;
+   samm:description "RFQ-Contact: BPDM/ or fill in by the customer"@en ;
+   samm:characteristic :SenderAdressCharacteristic ;
+   samm:exampleValue "Sunstreet 1, 5555 Sunstate" .
+
+:senderCompanyName a samm:Property ;
+   samm:preferredName "sender company name"@en ;
+   samm:description "RFQ-Contact: BPDM/ or fill in by the customer"@en ;
+   samm:characteristic :SenderCompanyNameCharacteristic ;
+   samm:exampleValue "ManufactureEnterprise" .
+
+:senderEMail a samm:Property ;
+   samm:preferredName "sender eMail"@en ;
+   samm:description "RFQ-Contact: BPDM/ or fill in by the customer"@en ;
+   samm:characteristic :SenderEMailCharacteristic ;
+   samm:exampleValue "johndoe@sunny.com" .
+
+:senderPhoneNumber a samm:Property ;
+   samm:preferredName "sender phone number"@en ;
+   samm:description "RFQ-Contact: BPDM/ or fill in by the customer"@en ;
+   samm:characteristic :SenderPhoneNumberCharacteristic ;
+   samm:exampleValue "555 123456" .
+
+:senderAccountAdress a samm:Property ;
+   samm:preferredName "sender account adress"@en ;
+   samm:description "RFQ-Contact: BPDM/ or fill in by the customer"@en ;
+   samm:characteristic :SenderAccountAdressCharacteristic ;
+   samm:exampleValue "Accountstreet 1, 1234 Accountstate" .
+
+:senderDeliveryAdress a samm:Property ;
+   samm:preferredName "sender delivery adress"@en ;
+   samm:description "RFQ-Contact: BPDM/ or fill in by the customer"@en ;
+   samm:characteristic :SenderDeliveryAdressCharacteristic ;
+   samm:exampleValue "Mystreet 1, 1234 Mystate" .
+
+:deliveryRequirements a samm:Property ;
+   samm:preferredName "delivery requirements"@en ;
+   samm:description "RFQ-Contact: filled by customer"@en ;
+   samm:characteristic :DeliveryRequirementsCharacteristic ;
+   samm:exampleValue "no plastic for packaging" .
+
+:BpnTrait a samm-c:Trait ;
+   samm:preferredName "BPN Business Partner Number Trait"@en ;
+   samm-c:baseCharacteristic :BpnCharacteristic ;
+   samm-c:constraint :BpnConstraint .
+
+:ContextCharacteristic a samm:Characteristic ;
+   samm:preferredName "context characteristic"@en ;
+   samm:dataType xsd:string .
+
+:CatenaXIdTraitCharacteristic a samm-c:Trait ;
+   samm:description "Trait to ensure UUID v4 data format"@en ;
+   samm-c:baseCharacteristic :Uuidv4Characteristic ;
```

Comment:
you can make use of shared aspect model (io.catenax.shared.uuid:1.0.0) for UUID.
you can refer this https://github.com/eclipse-tractusx/sldt-semantic-models/blob/a07bdd5b1515767d4f8d07deb47cf2a4aa728bef/io.catenax.quality_message_content/1.0.0/QualityMessageContent.ttl#L35C9-L35C17